app/controllers/export/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Export::BaseController < PublicFacingController
+  skip_before_action :set_cache_control_headers
+  skip_before_action :restrict_request_formats
+  before_action :require_export_data_permission!
+  before_action :set_api_cache_control_headers
+  before_action :set_api_access_control_allow_origin_headers
+  respond_to :json
+
+private
+
+  def require_export_data_permission!
+    forbidden! unless current_user.can_export_data?
```

Comment:
I'm a bit confused (I probably missed something) but I don't think this controller authenticates so wondering whether this blows up if current_user is nil or if this represents a different object when not authenticated.

Reply:
I've changed `Export::BaseController` to `Admin::Export::BaseController`, which will inherit `Admin::BaseController` which has authentication.

Will leave the check for the export data permission in place.